packages/grafana-data/src/types/config.ts

```diff
@@ -132,4 +133,6 @@ export interface GrafanaConfig {
   dateFormats?: SystemDateFormatSettings;
   sentry: SentryConfig;
   customTheme?: any;
+  azureCloud?: AzureCloud;
```

Comment:
Don't think you'll need this here since it's already available in grafana/runtime.

Reply:
OK removed. Please look at the GrafanaBootConfig, let me know what is the best place to leave the [AzureSettings](https://github.com/grafana/grafana/blob/6787a36b689653f0a4df4b52213eac528fed0dfb/packages/grafana-runtime/src/config.ts#L16-L19) interface. I need it to group the settings.